src/main/java/controller/AttendanceController.java

```diff
@@ -0,0 +1,120 @@
+package controller;
+
+import dto.DismissalCrewDto;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import model.AttendanceType;
+import model.Campus;
+import model.Command;
+import model.Crew;
+import model.Crews;
+import model.SubjectType;
+import util.StringParser;
+import util.TimeFormatter;
+import view.InputView;
+import view.ResultView;
+
+public class AttendanceController {
+
+    private final InputView inputView;
+    private final ResultView resultView;
+    private final Campus campus;
+
+    public AttendanceController(final InputView inputView, final ResultView resultView, final Campus campus) {
+        this.inputView = inputView;
+        this.resultView = resultView;
+        this.campus = campus;
+    }
+
+    public void start(final Crews crews) {
+        Command command = Command.from(inputView.readCommand(getTodayDate()));
+        if (command.equals(Command.QUIT)) {
+            return;
+        }
+        process(crews, command);
+        start(crews);
+    }
+
+    public static LocalDate getTodayDate() {
+        return LocalDate.of(2024, 12, 13);
+    }
+
+    private void process(final Crews crews, final Command command) {
+        processAttendance(crews, command);
+        if (command.equals(Command.CHECK_ATTENDANCE_BY_CREW)) {
+            checkAttendanceHistoryByCrew(crews);
+        }
+        if (command.equals(Command.CHECK_DISMISSAL_CREW)) {
+            checkDismissalCrews(crews);
+        }
+    }
+
+    private void processAttendance(Crews crews, Command command) {
```

Comment:
```suggestion
    private void processAttendance(final Crews crews, final Command command) {
```
전체적으로 컨벤션을 하나로 통일해보아요 👀

Reply:
앗.. 놓친 부분이 있었군요...!
피드백 감사합니다!
덕분에 찾아보다가 인텔리제이의 코드 자동완성 기능에 대해 알게 되어서 이제 자동으로 final이 붙도록 설정했어요 👍